examples/nextjs-shadcn/src/app/signer/route.ts

```diff
@@ -0,0 +1,113 @@
+import { mnemonicToAccount } from "viem/accounts";
+import axios from "axios";
+import { NextResponse, NextRequest } from "next/server";
+
+if (typeof process.env.FARCASTER_DEVELOPER_MNEMONIC === "undefined") {
+  throw new Error("FARCASTER_DEVELOPER_MNEMONIC is not defined");
+}
+const FARCASTER_DEVELOPER_MNEMONIC = process.env.FARCASTER_DEVELOPER_MNEMONIC;
+
+if (typeof process.env.FARCASTER_DEVELOPER_FID === "undefined") {
+  throw new Error("FARCASTER_DEVELOPER_FID is not defined");
+}
+const FARCASTER_DEVELOPER_FID = process.env.FARCASTER_DEVELOPER_FID;
+
+export async function GET(req: NextRequest, res: NextResponse) {
+  // Do whatever you want
+  try {
+    const response = await axios.get(
```

Comment:
is there a reason why axios is used over fetch here?

Reply:
copy pasta from neynar signer example - was tempted to rewrite but couldn't be bothered